Packs/PrismaCloudCompute/Scripts/CreatePrismaCloudComputeComplianceReportButton/README.md

```diff
@@ -0,0 +1,41 @@
+Generate a compliance report via button.
+
+## Script Data
+
+---
+
+| **Name** | **Description** |
+| --- | --- |
+| Script Type | python3 |
+| Cortex XSOAR Version | 6.8.0 |
+
+## Inputs
+
+---
+
+| **Argument Name** | **Description** |
+| --- | --- |
+| table | The table of data. |
```

Comment:
see my comment in the yml.

Reply:
I think generally a `table` in xsoar refers to a list of objects.  But maybe I can add a note to that effect to make it more clear?

Something like `The table of data (list of objects).` ??